app/models/review.rb

```diff
@@ -3,7 +3,7 @@ class Review < ApplicationRecord
   belongs_to :user
   has_many :tags, dependent: :destroy
   has_many :categories, through: :tags
-  # belongs_to :trail
+  belongs_to :trail
```

Comment:
Can delete this line? It's causing the seeding to break because this means we have to specify a trail when creating a review.